docs/source/develop.rst

```diff
@@ -97,6 +97,10 @@ For development, Dask uses the following additional dependencies::
 
    pip install pytest moto mock
 
+And the following dependencies are required to get all tests to pass::
+
+   conda install graphviz python-graphviz ipython
```

Comment:
I think that this is the intent of the `pip install` line just above.  Maybe roll those dependencies into here as well so that we don't ask users two do two things when they could do just one.

Reply:
Also, I think that python-graphviz brings in graphviz.

Alternatively, we might add `pytest.importorskip` lines in tests that require graphviz.  I don't think that people should have to have tools like graphviz or ipython to run our tests successfully.

Reply:
> Alternatively, we might add pytest.importorskip lines in tests that require graphviz.

This is my preference as well. I fixed a few of these recently, but there's still a few more it seems. The tests should pass fine with just:

```python
# in a clean python environment
pip install -e .[complete]
pip install pytest

py.test dask
```

Any differences from this is a bug in our test suite.

Reply:
Cool, thanks. I'll revert this part of the patch, and figure out instead how to make ipython, etc. optional for the tests.